cmd/update-main.go

```diff
@@ -235,20 +257,22 @@ func getDownloadURL() (downloadURL string) {
 	return minioReleaseURL + "minio"
 }
 
-func getUpdateInfo(timeout time.Duration, mode string) (older time.Duration, downloadURL string, err error) {
-	currentReleaseTime, err := GetCurrentReleaseTime()
+func (u *updateParams) getUpdateInfo(timeout time.Duration) (
```

Comment:
Changed it to not depend on the struct - since the struct can be initialized here. No need to export I think unless we are following a particular convention.